packages/odyssey-react-mui/src/Toast.tsx

```diff
@@ -95,7 +95,9 @@ const Toast = forwardRef(
     return (
       <Snackbar
         open={isVisible}
-        autoHideDuration={isDismissable ? undefined : autoHideDuration}
+        autoHideDuration={
+          isDismissable || autoHideDuration <= 0 ? undefined : autoHideDuration
+        }
```

Comment:
Prettier might remove these parens, but they make it easier to not break this logic:

```suggestion
        autoHideDuration={
          isDismissable || (autoHideDuration <= 0 ? undefined : autoHideDuration)
        }
```

Reply:
Wouldn't it be `(isDismissable || autoHideDuration <= 0) ? undefined : autoHideDuration`

Reply:
The intention here is:
- If `isDismissable`, autoHideDuration is undefined
- If `autoHideDuration <= 0`, autoHideDuration is undefined (ie, 0 and negative values mean it doesn't auto-hide)
- If neither, autoHideDuration is the autoHideDuration